pkg/apis/pipeline/v1beta1/taskrun_validation.go

```diff
@@ -75,6 +75,13 @@ func (ts *TaskRunSpec) Validate(ctx context.Context) (errs *apis.FieldError) {
 	errs = errs.Also(validateParameters(ts.Params).ViaField("params"))
 	errs = errs.Also(validateWorkspaceBindings(ctx, ts.Workspaces).ViaField("workspaces"))
 	errs = errs.Also(ts.Resources.Validate(ctx).ViaField("resources"))
+	if cfg.FeatureFlags.EnableDebug {
```

Comment:
Once this PR is using the "alpha" feature guard, see https://github.com/tektoncd/pipeline/blob/main/pkg/apis/pipeline/v1beta1/task_validation.go#L132 for an example where we validate that the alpha flag is set for a given feature.

Reply:
Thanks for the review, I will update this right away.

Reply:
Updated